{
  try {
    doInit();
  }
 catch (  Throwable t) {
    LOG.error(t);
  }
}
